test/upload-test-images.sh

```diff
@@ -17,6 +17,7 @@
 set -o errexit
 
 export GO111MODULE=on
+export ARCH=`uname -m | sed 's/x86_64/amd64/'`
```

Comment:
I think we should be careful about this. For some people like me the build machine is different from the deployment machine. So it might make more sense to make this an env var instead.

Reply:
good point, thank you for this reminder. I've updated the code actually to use env variable if target platform is not `linux/amd64`